2024-01-10 02:15:40 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 4 seconds before continuing (required by "messages.SendMessage")
2024-01-10 02:16:20 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
2024-01-10 16:07:46 - root - ERROR - Failed to initialize for user [5586931760]
Traceback (most recent call last):
  File "c:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\PyroEdgeGptBot.py", line 1683, in set_suggest_mode_handle
    EDGES[update.from_user.id]["bot"] = await EdgeGPT.Chatbot.create(cookies=BING_COOKIE)
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\EdgeGPT.py", line 42, in create
    await Conversation.create(self.proxy, cookies=cookies),
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\conversation.py", line 112, in create
    raise Exception("Authentication failed")
Exception: Authentication failed
2024-01-10 16:08:11 - root - ERROR - Failed to initialize for user [5586931760]
Traceback (most recent call last):
  File "c:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\PyroEdgeGptBot.py", line 1683, in set_suggest_mode_handle
    EDGES[update.from_user.id]["bot"] = await EdgeGPT.Chatbot.create(cookies=BING_COOKIE)
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\EdgeGPT.py", line 42, in create
    await Conversation.create(self.proxy, cookies=cookies),
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\conversation.py", line 112, in create
    raise Exception("Authentication failed")
Exception: Authentication failed
2024-01-10 16:15:22 - root - ERROR - Failed to initialize for user [5586931760]
Traceback (most recent call last):
  File "c:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\PyroEdgeGptBot.py", line 1683, in set_suggest_mode_handle
    EDGES[update.from_user.id]["bot"] = await EdgeGPT.Chatbot.create(cookies=BING_COOKIE)
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\EdgeGPT.py", line 42, in create
    await Conversation.create(self.proxy, cookies=cookies),
  File "C:\Users\CSR\Downloads\AdvChatGpt-main\AdvChatGpt-main\.venv\lib\site-packages\EdgeGPT\conversation.py", line 112, in create
    raise Exception("Authentication failed")
Exception: Authentication failed
2024-01-10 17:31:23 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.SendMessage")
2024-01-10 17:32:02 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
2024-01-10 17:34:55 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.SendMessage")
2024-01-10 17:35:40 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
2024-01-10 17:36:15 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.SendMessage")
2024-01-10 17:36:22 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.SendMultiMedia")
2024-01-10 17:36:51 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
2024-01-10 17:37:03 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 10 seconds before continuing (required by "messages.EditMessage")
2024-01-10 17:37:17 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
2024-01-10 17:37:47 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.SendMessage")
2024-01-10 17:38:38 - pyrogram.session.session - WARNING - [AdvChatGptTrialsX] Waiting for 3 seconds before continuing (required by "messages.EditMessage")
